Replicator/IncomingRev.cc

```diff
@@ -218,6 +221,7 @@ namespace litecore { namespace repl {
         // First create a Fleece document:
         Doc fleeceDoc;
         C4Error err = {};
+        bool didApplyDelta;
```

Comment:
initializae

Reply:
Booleans are already initialized (so are all integral types).
Just structs, classes, etc are uninitialized

Reply:
I doubt it. At least GPT does not think so.

Reply:
Oh I see, Xcode doesn't like it regardless of that.